freeclimb-cs-sdk/api/phonenumber/AvailablePhoneNumbersSearchFilter.cs

```diff
@@ -15,12 +15,21 @@ namespace com.freeclimb.api.phonenumber
     [JsonObject]
     public class AvailablePhoneNumbersSearchFilter
     {
-        [JsonProperty(PropertyName = "alias")]
-        private string alias = string.Empty;
-
         [JsonProperty(PropertyName = "phoneNumber")]
         private string phoneNumber = string.Empty;
 
+        [JsonProperty(PropertyName = "country")]
+        private string country = string.Empty;
+
+        [JsonProperty(PropertyName = "region")]
+        private string region = string.Empty;
+
+        [JsonProperty(PropertyName = "smsEnabled")]
+        private bool smsEnabled = false;
```

Comment:
on line 155 you check if `smsEnabled != null`, should this value be instantiated as `null` instead of `false` here?

Reply:
I instantiated both smsEnabled and voiceEnabled booleans as false by default because thats how the active property was added. Unsure if the same logic is to be used with these two or if they should be null be default. After looking at the doc, it doesn't specify what the default values should be so unsure.